horace_core/sqw/@axes_block/private/calc_bin_nodes_.m

```diff
@@ -49,8 +55,13 @@
     iint_ax = num2cell(obj.iint',2);
     axes(obj.iax) = iint_ax(:);
     npoints_in_axes = obj.nbins_all_dims+1;
+    %iax = false(4,1);
```

Comment:
Debugging statements?

Reply:
not sure what version should prevail. Left for clarity.